### PR TITLE
feat: show principal and address on hardware wallet

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

On account detail page, user should be able to call a function that displays principal and address on the hardware wallet.

# Changes

- fix ledger identity functions' callback that need `this` at runtime
- add `showAddressAndPubKeyOnHardwareWallet` to service
- create a component that contains a `button` to execute above function

### CODE REVIEW

File: frontend/svelte/src/routes/Wallet.svelte [hunk 1]
```diff
@@ -48,7 +49,11 @@
     <svelte:fragment slot="header">{$i18n.wallet.title}</svelte:fragment>
 
     {#if mainAccount}
-      <section>TBD - TODO(L2-429)</section>
+      <section>
+        <h1>TBD - TODO(L2-429)</h1>
+
+        <HardwareWalletShowAction />
```
Comment (mstrasinskis) [inline frontend/svelte/src/routes/Wallet.svelte:55]: Just curious, shouldn't there be something like {#if isHardwareWallet .... ?

**Review by mstrasinskis — APPROVED** [review]
LGTM, one question?

File: frontend/svelte/src/routes/Wallet.svelte (hunk 1 above)
Comment (peterpeterparker) [reply frontend/svelte/src/routes/Wallet.svelte:55]: Yes, definitely. There will be another action for the hardware wallet "Show neurons" and one for "Rename subaccount" so my idea is to create later an "AccountActionToolbar" component or something in which we will implement the logic you suggested. Just thought we can do that in a separate PR afterwards.

Comment (peterpeterparker) [reply frontend/svelte/src/routes/Wallet.svelte:55]: i'll do it in my next pr for "Rename subaccount"

**Pull request merged**